src/wallet/wallet.h

```diff
@@ -337,6 +335,7 @@ class CWalletTx : public CMerkleTx
     mutable CAmount nImmatureWatchCreditCached;
     mutable CAmount nAvailableWatchCreditCached;
     mutable CAmount nChangeCached;
+    mutable bool fInMempool;
```

Comment:
Moved it up. At some point someone can go through and mark all the bools as 1 bit, I'd think.